lambdas/ecs-events-to-slack/notify.py

```diff
@@ -24,7 +24,7 @@ def message_formatter(event, region):
 
     event_detail = event["detail"]
     task_arn = event_detail["taskArn"]
-    task_id = task_arn.split("/")[1]
+    task_id = task_arn.split("/")[2]
```

Comment:
Can you add a comment to this line with the format that `task_arn` is in, so the next person has a shot at debugging :-)